src/kili/presentation/client/cloud_storage.py

```diff
@@ -499,10 +497,10 @@ def update_data_integration(
         """
         return CloudStorageUseCases(self.kili_api_gateway).update_data_integration(
             name=name,
-            data_integration_id=DataIntegrationId(data_integration_id),
+            cloud_storage_integration_id=DataIntegrationId(cloud_storage_integration_id),
```

Comment:
for use cases, gateway, you can keep the "data integration, data connection"

we only use "cloud storage" for doc and user facing methods

Reply:
Yes but in this file cloud_storage_integration_id was used everywhere so we decided to use it as well for keep consistency, is it ok for you ?

Reply:
I have update the code for be more clear